zad3/app/controllers/bestiary_controller.rb

```diff
@@ -1,12 +1,14 @@
 # frozen_string_literal: true
 
 class BestiaryController < ApplicationController
-  def index
+  before_action :monsters
 
+  def index
   end
 
   def novigrad_monsters
-
+    @monsters = @monsters.select { |monster| monster[:location] == 'Novigrad' }
+    render 'index'
```

Comment:
Sprytnie 👍 🎉 Jak podszedłbyś do takiego nowego wymagania?:
- Strona z potworami z Novigradu powinna dodatkowo, nad tabelą, wyświetlać info o tym, że są to potwory z Novigradu

Reply:
Okej, tak też można 👍 🌮 Chodziło mi bardziej o to, żeby pozwolić railsom na użycie oddzielnego widoku dla `novigrad_monsters`, a dopiero w środku użyć [partiala](https://guides.rubyonrails.org/layouts_and_rendering.html#using-partials), który zajmuje się renderowanie tabeli z dowolnymi potworami. Tego samego partiala można wtedy użyć w `index.html.erb`. Takie podejście znacznie ułatwia wprowadzanie jakichkolwiek zmian 😉